spec/factory_girl/authorizations.rb

```diff
@@ -0,0 +1,8 @@
+FactoryGirl.define do
+  factory :authorization do
+    user nil
+provider "MyString"
```

Comment:
Inconsistent indentation detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.